server/routes/receiveData/index.js

```diff
@@ -1,10 +1,487 @@
 var router = require('express').Router();
 
+var Preamble = require('./../../database/models/preamble.js');
+var Raw = require('./../../database/models/raw.js');
+var ErrorInfo = require('./../../database/models/errorInfo.js');
+
+var CurrBatteryChargingAnalogVoltage = require('./../../database/models/currentData/battery/currBatteryChargingAnalogVoltage.js')
```

Comment:
can you switch all these to use the common functions instead of the explicit ones on the DB